redbox/models/file.py

```diff
@@ -20,23 +21,45 @@ class ProcessingStatusEnum(str, Enum):
     complete = "complete"
 
 
+class ContentType(str, Enum):
+    EML = "eml"
+    HTML = "html"
+    JSON = "json"
+    MD = "md"
+    MSG = "msg"
+    RST = "rst"
+    RTF = "rtf"
+    TXT = "txt"
+    XML = "xml"
+    JPEG = "jpeg"  # Must have tesseract installed
+    PNG = "png"  # Must have tesseract installed
+    CSV = "csv"
+    DOC = "doc"
+    DOCX = "docx"
+    EPUB = "epub"
+    ODT = "odt"
+    PDF = "pdf"
+    PPT = "ppt"
+    PPTX = "pptx"
+    TSV = "tsv"
+    XLSX = "xlsx"
+
+
 class File(PersistableModel):
-    path: str
-    type: str
-    name: str
+    path: str = Field(description="location of file")
+    type: ContentType = Field(description="content_type of file")
+    name: str = Field(description="file name")
     storage_kind: str = "local"
-    text: str = ""
-    processing_status: ProcessingStatusEnum = ProcessingStatusEnum.uploaded
+    text: Optional[str] = Field(description="file content", default=None)
```

Comment:
we could set this to be a required field if we had an `IngestQueueItem` to reference the file rather than reusing this class?